src/libs/ReportUtils.ts

```diff
@@ -1711,6 +1711,13 @@ function getDisplayNamesWithTooltips(
         });
 }
 
+/**
+ * Return the concatenated string of the display name of a list user
+ */
+function getUserDetailsTooltipText(accountIDs: number[]): string {
+    return accountIDs.map((accountID) => getDisplayNameForParticipant(accountID)).join(', ');
```

Comment:
NAB, but out of curiosity, would this work in TS? Maybe once it's converted to JS it does the same thing?
```js
return accountIDs.map(getDisplayNameForParticipant).join(', ');
```

Reply:
Which error? just curious if you know off the top of your head, otherwise I can try it myself. I thought it would work and the second param would simply be `undefined`